tests/test-services/src/kernel/manager.spec.ts

```diff
@@ -203,7 +221,7 @@ describe('kernel/manager', () => {
         const kernel = await manager.startNew();
         const emission = testEmission(manager.runningChanged, {
           test: () => {
-            expect(kernel.isDisposed).to.equal(false);
+            expect(kernel.isDisposed).to.equal(true);
```

Comment:
Note to self: this may need to be changed back to false after my recent manager updates.